purchase_stock_bl_date/readme/USAGE.rst

```diff
@@ -0,0 +1,11 @@
+* Set B/L Date in the purchase order as appropriate. When the purchase order is confirmed and
+  the receipt picking is generated, the set date is passed over to the corresponding field of the picking.
+* Adjust B/L Date of the receipt picking as necessary before validating the receipt.
+
+In case you forget to assign or need to change the B/L Date after the receipts have been
+validated, follow these steps:
+
+* Open the Receipts form and click on the "Unlock" button.
+* Enable the 'Force B/L Date' option and add or update the B/L Date.
+* After updating, click on the "Lock" button to update the inventory valuation and
+  the corresponding Stock Journal Entry.
```

Comment:
@AungKoKoLin1997 , Regarding this concern, is there any way to do checking whether the stock of the product that received on the transfer already shipped out or not? i guess it will be hard..
but if there's any way to do that, i think we shall add that checking and block the B/L dates update if the stock has been shipped out with user error and give information to manually adjust the related delivery to be able to update the B/L date. kindly your advise.